### ISSUE TITLE
utils.isEmpty() is useless and unnecessary

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1100]
_Imported from trac ticket 1100.
Created by **kbower**
Opened in trac: Sep 27, 2011, 6:47:48 PM
Last modified in trac: Oct 19, 2011, 4:53:18 PM_
***
Boolean("") evaluates to false all by itself.

```java
var kb:String;
if (kb) {
	trace("a. kb is true " + Boolean(kb));					
}
else {
	trace("a. kb is false " + Boolean(kb));					
}
kb = new String();
if (kb) {
	trace("b. kb is true " + Boolean(kb));					
}
else {
	trace("b. kb is false " + Boolean(kb));					
}
kb = '';
if (kb) {
	trace("c. kb is true " + Boolean(kb));					
}
else {
	trace("c. kb is false " + Boolean(kb));					
}
kb = 'kb';
if (kb) {
	trace("d. kb is true " + Boolean(kb));					
}
else {
	trace("d. kb is false " + Boolean(kb));					
}

var mb:*;
if (mb) {
	trace("a. mb is true " + Boolean(mb));					
}
else {
	trace("a. mb is false " + Boolean(mb));					
}
mb = new String();
if (mb) {
	trace("b. mb is true " + Boolean(mb));					
}
else {
	trace("b. mb is false " + Boolean(mb));					
}
mb = '';
if (mb) {
	trace("c. mb is true " + Boolean(mb));					
}
else {
	trace("c. mb is false " + Boolean(mb));					
}
mb = 'mb';
if (mb) {
	trace("d. mb is true " + Boolean(mb));					
}
else {
	trace("d. mb is false " + Boolean(mb));					
}
``````
a. kb is false false
b. kb is false false
c. kb is false false
d. kb is true true
a. mb is false false
b. mb is false false
c. mb is false false
d. mb is true true
```

**Comment by kbower** [comment]
_**kbower** trac comment on Oct 19, 2011, 4:53:18 PM:_

(Changeset [[1861]](../commit/335d112c09c96a8c2e8daf5cbf8b207010ec54f8))
 * search and remove all `utils.isEmpty()` references (closes #1100)

**Comment by kbower** [comment]
_**kbower** trac status change on Oct 19, 2011, 4:53:18 PM:_ **closed**
